linera-service/tests/end_to_end_tests.rs

```diff
@@ -569,17 +567,21 @@ impl TestRunner {
         path.into_os_string().into_string().unwrap()
     }
 
-    async fn generate_initial_validator_config(&self) {
+    async fn generate_initial_validator_config(&self) -> anyhow::Result<Vec<String>> {
         let mut command = self.command_for_binary("server").await;
         command.arg("generate").arg("--validators");
         for i in 1..=self.num_initial_validators {
             command.arg(&self.configuration_string(i));
         }
-        command
+        let output = command
             .args(["--committee", "committee.json"])
-            .status()
-            .await
-            .unwrap();
+            .stdout(Stdio::piped())
+            .spawn()?
+            .wait_with_output()
+            .await?;
```

Comment:
I would `.unwrap` or `.expect` here for consistency + it gives better stack traces.